### DISCUSSION TITLE
Import PPD file

### DISCUSSION THREAD

**Discussion by YinHana0505** [pyPhotometry/code#8] [Q&A] [answered]
Hi Thomas,

I have used pyphotometry to collect some data in the form of ppd.file. However I am having some problem in opening the file for analysis. I tried to use the code you provided but I keep encountering error like this:

![image](https://user-images.githubusercontent.com/85520531/234648348-15ec1401-deee-4506-9a19-d4514ae9dfcf.png)

I am still quite new to these so I am not sure what is the problem as I followed your guide on https://pyphotometry.readthedocs.io/en/latest/user-guide/importing-data/,  I hope you can help me. Could you advise me on how to open the file?

Thank you for your assistance and advice.

Warm regards
Hana

**Comment by ThomasAkam** [comment, marked as answer]
Hi Hana,

This error is happening because the `data_import` module is not located anywhere where Python is looking for it, so Python is failing to import it.  See [here](https://www.geeksforgeeks.org/where-does-python-look-for-modules/) for more information about where the python interpreter looks for modules.  The simplest way to fix this is to create a `code` folder for your project where you will locate the python files that you use for that project's analysis.  Then copy the file `data_import.py` from `pyPhotometry/tools` folder into the code folder.  When you open a python interpreter, you will then need to set the current working directory to the code folder you created.  If you use an IDE like [Spyder](https://www.spyder-ide.org/) you can do this using the folder browser in the GUI.  Alternatively you can get and set the current working directory using python commands.

```python
import os

os.getcwd() # Get the current working directory.

os.chdir('path//to//code//folder') # Set the current working directory.
```
You should then be able to import the `data_import` module.

best,

Thomas
